### ISSUE TITLE
Compat bounds

### ISSUE THREAD

**Issue by heliosdrm** [JuliaDynamics/RecurrenceAnalysis.jl#85]
In this moment all compat bounds in `Project.toml` are lower bounds. But the recommended policy for the General Registry is to include also upper bounds.

If we want to follow that recommendation, we should check what are the compatible lower and upper bounds of dependencies for the next release. According to the last successful CI check, the following package versions are compatible (if we consider the PR #84):

*   DelayEmbeddings 1.2.0
*   Distances 0.8.2
*   StaticArrays 0.12.1
*   UnicodePlots 1.1.0

The corresponding minor versions are safe upper bounds. Moreover, Distances 0.7 will *not* compatible after merging #84. So, the line for that package should be `Distances = "0.8".

It remains to be checked what lower bounds might be given for the other packages, if we want to provide a wider range of compatible versions.

(This also applies to the Julia versions: it works for any 1.x version, but I've not checked if it would still be compatlble with 0.7 - if we still want it to be compatible with it.)

**Comment by heliosdrm** [comment]
The smallest minor versions of the current dependencies registered in General (let aside `Distances`) are:

* DelayEmbeddings 0.1
* StaticArrays 0.8
* UnicodePlots 0.3

I have tested the package fixing those versions, and it passes (exact configuration below):

``` 
(RecurrenceAnalysis) pkg> st
Project RecurrenceAnalysis v1.0.4
    Status `D:\.julia\dev\RecurrenceAnalysis\Project.toml`
  [5732040d] DelayEmbeddings v0.1.1
  [b4f34e82] Distances v0.8.2
  [90137ffa] StaticArrays v0.8.3
  [b8865327] UnicodePlots v0.3.1
  [8bb1440f] DelimitedFiles
  [37e2e46d] LinearAlgebra
  [9a3f8284] Random
  [2f01184e] SparseArrays
  [10745b16] Statistics
```

I think that with this we can safely set also the lower compat bounds for the next release.

**Comment by heliosdrm** [comment]
Closed by #84
